Microsoft.Recognizers.Resources/CommonDefinitions.ttinclude

```diff
@@ -10,7 +10,7 @@
 <#@ import namespace="Microsoft.Recognizers.Resources.Common" #>
 <#
 	// Read the model file 
-	var input = new StringReader(File.ReadAllText(this.DataFilename));
+	var input = new StringReader(File.ReadAllText(Host.ResolveAssemblyReference("$(SolutionDir)") + this.DataFilename)); 

```

Comment:
@JuanAr , we can not generate code in our machine because of the path error. So we add this.